controller/iteration_test.go

```diff
@@ -454,11 +455,14 @@ func getChildIterationPayload(name *string) *app.CreateChildIterationPayload {
 	}
 }
 
-func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space.Space, area.Area, iteration.Iteration, iteration.Iteration) {
+// following helper function creates a space , root area, root iteration for that space.
+// Also creates a new iteration and new area in the same space
+func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space.Space, area.Area, iteration.Iteration, area.Area, iteration.Iteration) {
 	var spaceObj space.Space
```

Comment:
thanks @hectorj2f done in https://github.com/fabric8-services/fabric8-wit/pull/1466/commits/205b02ea0b2838beb01644bb2361117701df7f9f